src/resolve/resolved-dns-stub.h

```diff
@@ -25,6 +25,8 @@ struct DnsStubListenerExtra {
         union in_addr_union address;
         uint16_t port;
 
+        Set *refuse_record_types;
```

Comment:
this should not be part of the extra stub, but there should be a global field for this, i.e. in Manager objectwhere you defined the char* field right now

Reply:
@poettering added this by accident when trying to figure out where to put `Set`. I'll remove this shortly.